main.py

```diff
@@ -45,11 +44,15 @@ def __init__(self, data, namespace, name=None):
 
     @property
     def name(self):
+        log.warn(
+            "The name parameter will be deprecated soon.",
+            PendingDeprecationWarning
+        )
         return self._name
 
     @name.setter
     def name(self, x):
```

Comment:
Please, change "x" var to a most long name like "value".

Reply:
Done